### PR TITLE
LPS-141231 Add test CustomElementCanBeInstanceable

### PR DESCRIPTION
**Pull request by sergiojimcos**
**Test Scenario:**

Using JSON, add a Custom Element (Vanilla Counter) instanceable.
Using JSON, create a Content Page.
Add a Grid widget with two modules in the Content Page.
Display two Vanilla Counters in both modules
Check that Vanilla Counter can be instanceable.

**What new?**

RemoteApps.CustomElementCanBeInstanceable -> Test verifies that a Custom Element can be instanceable.

**JIRA Ticket**: https://issues.liferay.com/browse/LPS-141231

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase [hunk 1]
```diff
@@ -129,6 +129,70 @@ definition {
...
+			for (var n : list "1,2") {
+				PageEditor.addElementToColumn(
+					columnNumber = "${n}",
+					navTab = "Widgets",
+					portletName = "Vanilla Counter");
+
+				AssertElementPresent(
+					key_columnNumber = "${n}",
+					key_position = "1",
+					locator1 = "PageEditor#GRID_COLUMN");
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase:181]: I think I'd get rid of this assert... To go with the rule: one assert per test. Maybe there's some other way we can secure that this action is being performed correctly? (wait until the element is visible?)

**Comment by sergiojimcos** [comment]
ci:test:frontend:nocompile

**Review by magjed4289 — APPROVED** [review]
Looks good!

**Comment by sergiojimcos** [comment]
ci:forward

**Pull request closed without merging**